book/src/configuration.md

```diff
@@ -23,6 +23,15 @@ To override global configuration parameters, create a `config.toml` file located
 | `idle-timeout` | Time in milliseconds since last keypress before idle timers trigger. Used for autocompletion, set to 0 for instant. | `400` |
 | `completion-trigger-len` | The min-length of word under cursor to trigger autocompletion | `2` |
 | `auto-info` | Whether to display infoboxes | `true` |
+| `file-picker` | Sets options for file picker and global search. Multiple pairs in an inline table. Details below.  | `{hidden = true, parents = true, ignore = false, git-ignore = true, git-global = true, git-exclude = true } |
```

Comment:
I think it would be better to move this to an "`[editor.file-picker]` section" like the "`[editor]` section" above and have the same key, description, default table.

Reply:
Okay, I will make a commit for this today. Thank you!

Reply:
I think that the commit [80ba0e0](https://github.com/helix-editor/helix/pull/988/commits/80ba0e071f847cf549825e852d29fad2e62eca9c) has the changes you were looking for!